websrc/source/documents/tutorials/prometheus-tutorial.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: "documents"
+page_title: "Contiv Prometheus Integration for Kubernetes"
+sidebar_current: "prometheus_tutorial"
+description: |-
+  Contiv Prometheus Integration
+---
+
+
+
+
+## Data Analytics with Prometheus and Grafana for Kubernetes
+
+- [Setup](#setup)
+- [Chapter 1 - Prometheus and Grafana](#ch1)
+- [Chapter 2 - Basic Ping Example](#ch2)
+- [Resources](#resources)
+
+This walks through the setup for Prometheus and Grafana and an example that may be useful for troubleshooting connectivity issues within your cluster.
+
+### <a name="setup"></a> Setup
+
+Please finish the [Container Networking Tutorial](https://contiv.github.io/documents/tutorials/networking-kubernetes-16.html), but do not clean up your cluster.
```

Comment:
Please change this to be a relative link (i.e., `/documents/...`)

I don't anticipate the domain name changing any time soon (lol), but there's no point in having extra text here which could cause a broken link at same point in the future